src/helpdesk_proj/settings.py

```diff
@@ -8,7 +8,9 @@
 # This SECRET_KEY must be changed!
 SECRET_KEY = 'f867a25e39a5214669e06ba9e0d3bc94e6920c02d0159d67e05491ef6ecfd96d'
 DEBUG = True  # This should always be False in production!
-ALLOWED_HOSTS = ['localhost', '127.0.0.1', 'helpdesk-vht.herokuapp.com']
+ALLOWED_HOSTS = ['localhost', '127.0.0.1', 'helpdesk-vht.herokuapp.com', '44.204.81.89']
```

Comment:
Since we're using AWS instead of Heroku, can you remove Heroku as one of the allowd hosts?